concourse/scripts/spec/kibana_set_utc_spec.rb

```diff
@@ -0,0 +1,116 @@
+    end
+
+    @elasticsearch.get(KIBANA_CONFIG_PATH).returning('{ "found": false }', 404)
+    @elasticsearch.put(KIBANA_INDEX_PATH).returning('{}', 201)
+    @elasticsearch.put(KIBANA_CONFIG_PATH).returning('{}', 201)
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(0)
+    expect(@elasticsearch.received_requests.size).to be(3)
+    expect(@elasticsearch.received_requests).to contain_request('GET', KIBANA_CONFIG_PATH)
+    expect(@elasticsearch.received_requests).to contain_request('PUT', KIBANA_INDEX_PATH)
+    expect(@elasticsearch.received_requests).to contain_request('PUT', KIBANA_CONFIG_PATH)
+  end
+
+  it "adds utc config if index exists but config is not present" do
+    @elasticsearch.get(KIBANA_CONFIG_PATH).returning('{ "found": true }', 200)
+    @elasticsearch.put(KIBANA_CONFIG_PATH).returning('{}', 200)
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(0)
+
+    expect(@elasticsearch.received_requests.size).to be(2)
+    expect(@elasticsearch.received_requests).to contain_request('GET', KIBANA_CONFIG_PATH)
+    expect(@elasticsearch.received_requests).to contain_request('PUT', KIBANA_CONFIG_PATH)
+  end
+
+  it "adds utc config if index exists and wrong config is present" do
+    @elasticsearch.get(KIBANA_CONFIG_PATH).returning('{ "found": true, "_source": { "dateFormat:tz": "NOT_UTC" } }', 200)
+    @elasticsearch.put(KIBANA_CONFIG_PATH).returning('{}', 200)
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(0)
+
+    expect(@elasticsearch.received_requests.size).to be(2)
+    expect(@elasticsearch.received_requests).to contain_request('GET', KIBANA_CONFIG_PATH)
+    expect(@elasticsearch.received_requests).to contain_request('PUT', KIBANA_CONFIG_PATH)
+  end
+
+  it "reports an error if elastic search is not available" do
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(1)
+    expect(@elasticsearch.received_requests.size).to be(0)
+  end
+
+  it "does nothing if index exists and config is already UTC" do
+    @elasticsearch.get(KIBANA_CONFIG_PATH).returning('{ "found": true, "_source": { "dateFormat:tz": "UTC" } }', 200)
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(0)
+
+    expect(@elasticsearch.received_requests.size).to be(1)
+    expect(@elasticsearch.received_requests).to contain_request('GET', KIBANA_CONFIG_PATH)
+  end
+
+  it "reports an error if elastic search responds with an unexpected status code" do
+    @elasticsearch.get(KIBANA_CONFIG_PATH).returning('{ "found": true, "_source": { "dateFormat:tz": "UTC" } }', 500)
+
+    run_set_utc_script
+
+    expect(last_command_started).to have_exit_status(1)
+
+    expect(@elasticsearch.received_requests.size).to be(1)
+    expect(@elasticsearch.received_requests).to contain_request('GET', KIBANA_CONFIG_PATH)
+  end
+
+  RSpec::Matchers.define :contain_request do |expected_method, expected_path|
```

Comment:
Within a describe block, you can just do:
```ruby
matcher :contain_request do ...
```
This also scopes the matcher to the describe block.  I'm not sure whether the same happens when using `RSpec::Matchers.define`.